rtb/analisi-requisiti/src/requisiti.tex

```diff
@@ -75,10 +82,34 @@ \subsection{Requisiti funzionali}
      RFO54 & Il cliente deve poter aprire un canale di comunicazione tramite chat verso un ristoratore & UC49 \\ \hline
     \end{longtblr}
     \end{center}
+
+\subsubsection{Facoltativi}
+
+\begin{center}
+    \begin{tblr}{
+        colspec={|X[1.5cm]|X[3cm]|X[8cm]|X[2.5cm]|},
+        row{odd}={bg=white},
+        row{even}={bg=lightgray}
+        }
+        \hline
+
+        \textbf{Codice} & \textbf{Caratteristica} & \textbf{Descrizione} & \textbf{Fonte} \\
+
+         RFD3 & Sicurezza & Le comunicazioni tra il client e il server devono essere crittografate, data la natura sensibile dei dati inseriti dagli utenti (tramite SSL/TLS) & Capitolato \\ \hline
```

Comment:
da mettere SSL/TLS con apice riferito al glossario in cui diciamo di che si tratta?